### DISCUSSION TITLE
SEB failed to start a new session

### DISCUSSION THREAD

**Discussion by loichustinx** [SafeExamBrowser/seb-win-refactoring#692] [Q&A]
Can anyone point out the exact reason of the error based on the attached logs please?

<img width="171" alt="error" src="https://github.com/SafeExamBrowser/seb-win-refactoring/assets/115798692/cc60bb15-5b05-4ba1-8cad-fba3f6eca9ad">

[2023-09-15_14h46m36s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/13164025/2023-09-15_14h46m36s_Runtime.log)
[2023-09-15_14h46m36s_Service.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/13164026/2023-09-15_14h46m36s_Service.log)

**Comment by dbuechel** [comment]
You're using a configuration which has the SEB Service component enabled (which we in general do not recommend, especially on BYOD). Unfortunately, the SEB Service fails to perform all its operations, which in turn leads to the session start not being successful:

`2023-09-15 14:47:20.916 [06] - ERROR: Failed to start new service session within 30 seconds!`

The reason appears to be that the SEB Service cannot disable the Windows Update system service, which you appear to have forced via your configuration (see https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane > "Allow Windows Update to run while SEB is running"):

```
2023-09-15 14:46:56.549 [09] - ERROR: [WindowsUpdateConfiguration] Failed to set service 'wuauserv' => 'stopped'!

   Exception Message: Er is een time-out opgetreden en de bewerking is niet voltooid.
   Exception Type: System.ServiceProcess.TimeoutException
```

**Reply by loichustinx** [reply]
Thanks for the quick reply.

Unfortunatly this error occured after using a third-party tool to create the test. (Bookwidgets) So we can't use the SEB configuration tool ...

**Reply by dbuechel** [reply]
Hm, okay, then the Bookwidgets developers would either need to change something on their end to allow to optionally enable that setting or you'd need to change your setup so that you deliver a custom configuration according to your needs, which then loads Bookwidgets (without using their configuration, if that's even possible).
